packages/ra-core/src/actions/listActions.ts

```diff
@@ -8,7 +8,7 @@ export interface ListParams {
     page: number;
     perPage: number;
     filter: any;
-    displayedFilters: any;
+    displayedFilters?: { [key: string]: boolean } | null;
```

Comment:
Maybe we could extract a `DisplayedFilter` type which would be `Record<string, boolean>` ? It could be inside the root `types.ts` file? What do you think about it?